src/keplr/index.ts

```diff
@@ -1,28 +1,64 @@
 import { getAkashTypeRegistry } from "../stargate";
 import { defaultRegistryTypes, SigningStargateClient } from "@cosmjs/stargate";
 import { Registry } from "@cosmjs/proto-signing";
+import { AminoTypes } from "@cosmjs/stargate";
+import { Certificate } from "../protobuf/akash/cert/v1beta2/cert";
 
 export function getChains() {
   return {
-    mainnet: { id: "" },
-    testnet: { id: "akash-testnet-6" },
+    mainnet: {
+      id: "akashnet-2",
+      name: "Akash Mainnet",
+      messagePath: "v1beta2",
+    },
+    testnet: { id: "testnet-1", name: "Akash Testnet", messagePath: "v1beta2" },
   };
 }
 
 export function getSigner(chain: any) {
-  return (window as any).getOfflineSigner(chain.id);
+  return (window as any).getOfflineSignerAuto(chain.id);
 }
 
-export async function get(chain: any, signer: any) {
+export async function get(chain: any, signer: any, endPoint: string) {
+  const customAminoTypes: any = new AminoTypes({
+    "/akash.cert.v1beta2.MsgCreateCertificate": {
```

Comment:
might be nice to have a helper function for these like `akashProtoPath("MsgCreateCertificate")` given that we seem to like to change the version. That may not be the point of this PR however.

Reply:
I do have ref to that path somewhere in a constant, just have not inlined it here since I'm still working on getting this damn Amino to work, and wanted it to be obvious what this was doing.